_data/meltano/extractors/tap-rest-api-msdk/widen.yml

```diff
@@ -52,6 +52,25 @@ settings:
   kind: object
   label: AWS Credentials
   name: aws_credentials
+- description: 'The style of Backoff [message|header] applied to rate limited APIs.
+    Backoff times (seconds) come from response either the `message` or `header`.
+    Optional: Defaults to `None`.'
+  kind: string
+  label: Backoff Type
+  name: backoff_type
+  value: none
```

Comment:
@s7clarke10 similar to my comment in https://github.com/meltano/hub/pull/1576/files#r1385464176. Should these `value: none` settings be excluded?

Reply:
Quick question here. Should I even add value entries for defaults if tap-rest-api-msdk has defaults set in the tap. I'm starting to think that I should remove most of the value statements that I added (they are not really required).

Reply:
@s7clarke10 we're somewhat inconsistent on this across the hub. On one hand its nice for documenting the defaults on the plugin pages but on the other hand we could accidentally be overriding changes to the default in the tap if we dont also keep the hub definition up to date. So I generally feel fine including defaults for SDK based plugins and for non-SDK based plugins I only include them if we want to override the default in the tap (although its better to make actual code changes to the tap but sometimes thats not possible).

Reply:
Thanks for that feedback. On this basis I have just removed the none defaults.